runners/google-cloud-dataflow-java/examples/build.gradle

```diff
@@ -39,6 +39,8 @@ dependencies {
 def gcpProject = project.findProperty('gcpProject') ?: 'apache-beam-testing'
 def gcsTempRoot = project.findProperty('gcsTempRoot') ?: 'gs://temp-storage-for-end-to-end-tests/'
 def dockerImageName = project(':beam-runners-google-cloud-dataflow-java').ext.dockerImageName
+// If -PuseExecutableStage is set, the use_executable_stage_bundle_execution wil be enabled.
+def fnapiExperiments = project.hasProperty('useExecutableStage') ? 'beam_fn_api,use_executable_stage_bundle_execution' : "beam_fn_api"
```

Comment:
Is there a way to structure this logic such that it can be shared rather than duplicated? For example, I see that they already differ slightly in a way that might break (this one has no space between experiments, which the other does)

Reply:
That's in purpose since this feature is not stable enough to run tests automatically.